docs/inventory.md

```diff
@@ -0,0 +1,246 @@
+# NixOS project resource inventory
+
+This is the current list of hardware and services that everyone has access to.
+
+# Accounts
+## GitHub
+
+owner: Eelco, Rob, Domen, Garbas
+
+## Domains
+
+owner: Eelco
+* nixos.org - https://www.uniteddomains.com/
+
+## DNS
+
+owner: Eelco
+Managed manually be Eelco via the United Domains web UI
+
+## AWS account
+
+owner:  Infor
+alias: lb-nixos
+access: Rob and Eelco
+
+## Packet.net
+
+owner: Graham
+
+## Hetzner Cloud
+
+owner: Graham
+(for ofborg)
+
+## IRC logging bot
+
+owner: SamuelDR
+https://logs.nix.samueldr.com/nixos/
+
+nick:
+
+    {`-`}_
```

Comment:
The registered nickname is <code>{\`-\`}</code>, it sometimes restarts after network events (splits, ping timeouts) and will not recover its registered nickname (as there is no code to do so).

As its nickname may cause issues with markdown implementations, you can use this to inline it:

    <code>{\`-\`}</code>

Furthermore, while not publicly advertised, but publicly shared in the past, this is the overlay used for the IRC bot.

 * https://gitlab.com/samueldr.nix/overlays/irclogger

Reply:
fixed